assets/js/table-wrappers.js

```diff
@@ -0,0 +1,10 @@
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
can we add a layout change or a generater script modificatino instead of js that mutates the structure like this on page load?